webaudio/resources/audit.js

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// Wondering why audit.js exists? TL;DR: it's a legacy thing...
```

Comment:
This feels too aggressive. Reading that issue, there are valid concerns raised by the owners of audit.js that they feel testharness.js does not provide them. Let's just provide the link for people to read without making a claim on it being 'legacy'.

```suggestion
// See https://github.com/web-platform-tests/wpt/issues/12781 for information on
// the purpose of audit.js, and why testharness.js does not suffice.
```